airbyte-integrations/connectors/source-confluence/source_confluence/schemas/search.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": { "type": ["null", "integer"] },
+    "key": { "type": ["null", "string"] },
+    "name": { "type": ["null", "string"] },
+    "type": { "type": ["null", "string"] },
+    "status": { "type": ["null", "string"] },
+    "permissions": { "type": ["null", "array"] },
+    "icon": { "type": ["null", "object"] },
+    "description": { "type": ["null", "object"] },
+    "_expandable": { "type": ["null", "object"] },
+    "_links": { "type": ["null", "object"] }
```

Comment:
Please check https://docs.airbyte.com/connector-development/schema-reference
Missing additionalProperties for objects.